enterprise/app/code/code.tsx

```diff
@@ -56,14 +60,17 @@ export default class CodeComponent extends React.Component<Props> {
   state: State = {
     owner: "",
     repo: "",
+    commitSHA: "",
     repoResponse: undefined,
     treeShaToExpanded: new Map<string, boolean>(),
     treeShaToChildrenMap: new Map<string, any[]>(),
     treeShaToPathMap: new Map<string, string>(),
     fullPathToModelMap: new Map<string, any>(),
+    fullPathToDiffModelMap: new Map<string, any>(),
     originalFileContents: new Map<string, any>(),
     currentFilePath: "",
     changes: new Map<string, string>(),
+    mergeConflicts: new Map<string, string>(),
     pathToIncludeChanges: new Map<string, boolean>(),
```

Comment:
maps are the most powerful data structure known to man

Reply:
loves me some maps lol